components/browser/storage-sync/src/test/java/mozilla/components/browser/storage/sync/PlacesHistoryStorageTest.kt

```diff
@@ -1314,7 +1314,7 @@ class PlacesHistoryStorageTest {
     @Test
     fun `safe read from places`() = runBlocking {
         val result = history.handlePlacesExceptions("test", default = emptyList<HistoryMetadata>()) {
-            throw PlacesException("test")
+            throw PlacesException.UnexpectedPlacesException("test")
```

Comment:
Shouldn't this have been rethrown? The test was initially checking that non-panic errors are swallowed by readers. But `UnexpectedPlacesException` doesn't sound like something we should be catching.